src/freenas/usr/local/etc/syslog-ng.conf.freenas

```diff
@@ -200,3 +205,11 @@ log { source(src); filter(f_emerg); destination(allusers); };
 #
 log { source(src); filter(f_ppp); destination(ppp); };
 log { source(src); filter(f_mdnsresponder); filter(f_info); destination(mdnsresponder); };
+
+
+#
+# nginx.err                                             /var/log/nginx/error.log
+# nginx.info                                            /tmp/nginx-access.log
+#
+log { source(src); filter(f_nginx); filter(f_is_info); destination(nginx_access); };
+log { source(src); filter(f_nginx); filter(f_err); destination(nginx_error); };
```

Comment:
Don't we want to add an f_is_err filter and use it here so we don't mix accesslogs into error logs?